src/Symfony/Component/Messenger/Bridge/Redis/Transport/Connection.php

```diff
@@ -154,7 +154,7 @@ private static function initializeRedis(\Redis|Relay $redis, string $host, int $
      */
     private static function initializeRedisCluster(?\RedisCluster $redis, array $hosts, string|array|null $auth, array $params): \RedisCluster
     {
-        $redis ??= new \RedisCluster(null, $hosts, $params['timeout'], $params['read_timeout'], (bool) $params['persistent'], $auth, ...\defined('Redis::SCAN_PREFIX') ? [$params['ssl'] ?? null] : []);
+        $redis ??= new \RedisCluster(null, $hosts, $params['timeout'], $params['read_timeout'], isset($params['persistent_id']), $auth, ...\defined('Redis::SCAN_PREFIX') ? [$params['ssl'] ?? null] : []);
```

Comment:
Here I changed it to `isset($params['persistent_id'])` because `persistent` is not a default option and this option seems to be for me for this case. Otherwise I'd have do add the default option for `persistent`.

Reply:
`$params['persistent'] ?? false`?